ethereumj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -871,6 +880,23 @@ else if (oldValue != null && newValue.isZero()) {
                     program.step();
                 }
                 break;
+                case EXTCODEHASH: {
+                    DataWord address = program.stackPop();
+                    PrecompiledContracts.PrecompiledContract contract =
+                            PrecompiledContracts.getContractForAddress(address, blockchainConfig);
+
+                    byte[] codeHash;
+                    // The EXTCODEHASH of an precompiled contract is either c5d246... or 0
```

Comment:
I think it means that if precompile acc is non-existent the hash is 0 otherwise the hash is an empty data hash.

Reply:
Hence, precompile contract doesn't need a special case to be processed